libretroshare/src/dht/p3bitdht_peers.cc

```diff
@@ -242,7 +242,7 @@ int p3BitDht::addKnownPeer( const RsPeerId &pid,
 	addrv4.sin_addr = ap->sin_addr;
 	addrv4.sin_port = ap->sin_port;
 
-	int p3type = 0;
+	RsDhtPeerType p3type = static_cast<RsDhtPeerType>(0);
```

Comment:
This is surprising. Don't you have a value corresponding to 0? If not, add one named "UNKNOWN" and use it.

Reply:
Fixed, there is a value for 0, don't asked me why i haven't used it back then. I guess i tried to stick as close as possible to the previous code :man_shrugging: